amplipi/ctrl.py

```diff
@@ -418,6 +439,61 @@ def reinit(self, settings: models.AppSettings = models.AppSettings(), change_not
     # configure all of the groups (some fields may need to be updated)
     self._update_groups()
 
+  def toggle_persist_logs(self, value: Optional[bool] = None):
+    """
+    Toggles the option within journald to save logs to memory or storage.
+    Accepts an option bool called "value" for if you wish to change the value to a specific option rather than simply flip the self.persist_logs bool
+    """
+    try:
+      conf_path = '/etc/systemd'
+
+      config = configparser.ConfigParser(strict=False, allow_no_value=True)
+      config.read(f'{conf_path}/journald.conf')
+
+      if 'Journal' not in config:
+        config.add_section('Journal')
+
+      goal_value: bool
+      if value is not None:
+        goal_value = value
+      else:
+        goal_value = not self.persist_logs
```

Comment:
Added the option to explicitly manually toggle the persistence rather than always flicking the switch to the inverse of the present setting